web_accounts_list.json

```diff
@@ -2808,6 +2786,28 @@
          "known_accounts" : ["john","sammy"],
          "category" : "gaming",
          "valid" : true
-      }
+      },
+      {
+         "name" : "Fandom",
+         "check_uri" : "https://www.fandom.com/u/{account}",
+         "account_existence_code" : "200",
+         "account_existence_string" : "| Profile | Fandom",
+         "account_missing_string" : "Not Found",
+         "account_missing_code" : "404",
+         "known_accounts" : ["EJacobs94","Drew_Dietsch"],
+         "category" : "gaming",
+         "valid" : true
+      },
+      {
+         "name" : "Zillow",
+         "check_uri" : "https://www.zillow.com/profile/{account}/",
+         "account_existence_code" : "200",
+         "account_existence_string" : "- Reviews | Zillow",
+         "account_missing_string" : " Real Estate Agent Reviews | Zillow",
+         "account_missing_code" : "200",
+         "known_accounts" : ["awhelgeson","coosbayrealestate"],
+         "category" : "shopping",
+         "valid" : true
+      },

```

Comment:
this trailing comma will make this invalid JSON. I will accept the PR and then mod the file.